test/test_gpu_outer_reduction.cpp

```diff
@@ -2288,17 +2288,21 @@ TEST_F(OuterReductionTest, OuterReductionMagicScheduler) {
     auto options = at::TensorOptions()
                        .dtype(data_type_to_aten(dtype))
                        .device(at::kCUDA, 0);
-    auto t0 = at::randn(shape, options);
+    auto t0 = at::ones(shape, options);
     std::vector<c10::IValue> inputs({t0});
     FusionExecutorCache executor_cache(std::move(fusion));
     auto cg_outputs = executor_cache.runFusionWithInputs(inputs);
     auto ref = t0.to(at::kFloat).sum({0});
     testValidate(
         executor_cache.fusion(), cg_outputs, inputs, {ref}, __LINE__, __FILE__);
   };
+
+  test(2048, 32768);
```

Comment:
This is covered by the following loops I believe so you can remove before merging.

Reply:
thanks! Looks like I pushed some debug changes by accident.